server/Logic/PointValidator.cs

```diff
@@ -29,12 +29,9 @@ public static void Validate(Point point, Tag tag)
                 {
                     throw new ValidationException($"Property not found: {tagField.Name}");
                 }
-                var pointFieldType = point.Extra[tagField.Name].GetType();
-                if (pointFieldType != tagField.Type)
+                if (!tagField.Type.TypeIsValid(point.Extra[tagField.Name]))
```

Comment:
this one check fail to validate type of incoming time and date type, please see comments below in tags.json.example